src/domain/notifications/v2/notifications.repository.integration.spec.ts

```diff
@@ -543,12 +572,13 @@ describe('NotificationsRepositoryV2', () => {
         signer_address: authPayload.signer_address,
       });
 
-      await notificationsRepositoryService.deleteSubscription({
+      const result = notificationsRepositoryService.deleteSubscription({
         deviceUuid: upsertSubscriptionsDto.deviceUuid as UUID,
         chainId: upsertSubscriptionsDto.safes[0].chainId,
         safeAddress: upsertSubscriptionsDto.safes[0].address,
       });
 
+      await expect(result).rejects.toThrow();
```

Comment:
That’s intentional because this test is not designed to check the type of the error. There is a separate test earlier that handles that purpose:
`it('Should throw NotFoundException if a subscription does not exist', async () => {`